src/frontend/components/CardList.js

```diff
@@ -97,45 +111,46 @@ class CardList extends React.Component {
     connectDropTarget: PropTypes.func.isRequired,
     isOver: PropTypes.bool.isRequired,
     cardList: PropTypes.object.isRequired,
-    moveCard: PropTypes.func,
-    findCard: PropTypes.func,
   };
 
   render() {
     const { connectDropTarget, isOver } = this.props;
-    const { cardList, moveCard, findCard } = this.props;
-    const cards = cardList.cards.edges;
+
+    const { cardList } = this.props;
+
+    let cards = cardList.cards.edges.map(({node}) => node);
```

Comment:
I was thinking we can actually just get rid of this client sorting altogether for now, since the default sort of the data by cardListRank takes care of maintaining the sort order.